03_ScienceExamples/GalacticStructure/ExploringSMASHDR2.ipynb

```diff
@@ -2277,7 +2277,7 @@
     "ax1 = fig.add_subplot(121)\n",
     "ax2 = fig.add_subplot(122)\n",
     "\n",
-    "# color-magnitude diagram (Hess diagram)\n",
+    "# color-color diagram\n",
```

Comment:
"Hess diagram" could be put back with a link to the short wikipedia page: https://en.wikipedia.org/wiki/Hess_diagram